### PR TITLE
try peastman suggestion

### PR DESCRIPTION
**Pull request by cxhernandez**
re: https://github.com/omnia-md/conda-recipes/issues/457

### CODE REVIEW

**Comment by jchodera** [comment]
That's weird.  Appveyor py34 only is failing with
```
Error: unknown section: patches
 
ERROR: File does not exist: C:\Miniconda3\conda-bld\win-32\gpy-0.8.8-py34_0.tar.bz2
```

**Comment by cxhernandez** [comment]
I think I put `patches` on the wrong indentation level. Let's see if this works....

File: gpy/bld.bat [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/openmp
```
Comment (mpharrigan) [inline gpy/bld.bat]: you might want to get rid of this

Comment (cxhernandez) [inline gpy/bld.bat]: woops! forgot about that

**Comment by cxhernandez** [comment]
Not sure why it's looking for a tarball....
```
ERROR: File does not exist: C:\Miniconda3\conda-bld\win-32\gpy-0.8.8-py34_0.tar.bz2
```

**Comment by mpharrigan** [comment]
```
can't find file to patch at input line 5
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/setup.py b/setup.py
|index 0432cba..3185891 100644
|--- a/setup.py
|+++ b/setup.py
--------------------------
File to patch:  
Skip this patch? [y]  
Skipping patch.
2 out of 2 hunks ignored 
```

**Comment by mpharrigan** [comment]
It's not building it on py35
```
[conda-build-all] Scheduling the following builds (win-64):
  pyemma-2.1rc3-np19py35_0.tar.bz2
  pyemma-2.1rc3-np110py35_0.tar.bz2
Removing old build environment
Removing old work directory
BUILD START: pyemma-2.1rc3-np19py35_0
```

**Comment by mpharrigan** [comment]
or 27

**Comment by cxhernandez** [comment]
Getting the same error after trying to follow this [blog post](https://ariejan.net/2009/10/26/how-to-create-and-apply-a-patch-with-git/) on how to create a patch....

**Comment by mpharrigan** [comment]
why is it not building on anything except windows py34

**Comment by mpharrigan** [comment]
thought: it's trying to patch `setup.py` but its working directory is probably where the conda recipe is, so it should be looking for `../../setup.py` or something?

**Comment by cxhernandez** [comment]
Okay, it still seems to be failing for py34 on Win, so I'm doubtful that https://github.com/omnia-md/conda-recipes/issues/457#issuecomment-191972357 is the underlying issue.

**Comment by jchodera** [comment]
It looks like the `/openmp` option still isn't being added where it is needed:
```
c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\BIN\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -IC:\Miniconda3\envs\_build\lib\site-packages\numpy\core\include -I. -IC:\Miniconda3\envs\_build\include -IC:\Miniconda3\envs\_build\include /TcGPy/kern/src/stationary_cython.c /Fobuild\temp.win32-3.4\Release\GPy/kern/src/stationary_cython.obj -fopenmp -O3
cl : Command line warning D9002 : ignoring unknown option '-fopenmp'
cl : Command line warning D9002 : ignoring unknown option '-O3'
stationary_cython.c
c:\miniconda3\envs\_build\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION 
c:\miniconda3\conda-bld\work\gpy\kern\src\stationary_utils.h(2) : fatal error C1083: Cannot open include file: 'omp.h': No such file or directory 
error: command 'c:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\VC\\BIN\\cl.exe' failed with exit status 2
```

**Comment by cxhernandez** [comment]
Oh! I think it's that I'm not applying the patch correctly....

**Comment by cxhernandez** [comment]
> why is it not building on anything except windows py34

It seems like it's built successfully on every other platform:
https://anaconda.org/omnia/gpy/files

**Comment by jchodera** [comment]
Go ahead and merge this if the additions have helped some of the non py34 platforms. Not sure what is going on with py34

**Comment by cxhernandez** [comment]
The non-py34 builds completed just fine from the initial PR, so there's no reason to add this patch unless it fixes py34.

**Comment by mpharrigan** [comment]
>  It seems like it's built successfully on every other platform:
> https://anaconda.org/omnia/gpy/files

Oh! screw py34... add a "skip" entry

**Pull request closed without merging**